### ISSUE TITLE
API Stability: Stick Plots

### ISSUE THREAD

**Issue by jamesprayner** [DFO-Ocean-Navigator/Ocean-Data-Map-Project#679]
**Describe the bug**
Just like virtual mooring plots, by default the point window component will choose `NaN undefined NaN` as a start time without any user input.

However, manually creating a query and requesting gives the exact same error as a query missing starttime:
```
Traceback (most recent call last):
  File "/home/buildadm/miniconda/3/ubuntu/16.04/amd64/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/buildadm/miniconda/3/ubuntu/16.04/amd64/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/home/buildadm/miniconda/3/ubuntu/16.04/amd64/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/buildadm/miniconda/3/ubuntu/16.04/amd64/envs/navigator/lib/python3.6/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/home/buildadm/miniconda/3/ubuntu/16.04/amd64/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/buildadm/miniconda/3/ubuntu/16.04/amd64/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/buildadm/miniconda/3/ubuntu/16.04/amd64/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/buildadm/miniconda/3/ubuntu/16.04/amd64/envs/navigator/lib/python3.6/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/home/buildadm/miniconda/3/ubuntu/16.04/amd64/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/buildadm/miniconda/3/ubuntu/16.04/amd64/envs/navigator/lib/python3.6/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/buildadm/Ocean-Data-Map-Project/routes/api_v1_0.py", line 549, in plot_v1_0
    img, mime, filename = plotter.run()
  File "/home/buildadm/Ocean-Data-Map-Project/plotting/plotter.py", line 58, in run
    _ = self.prepare_plot()
  File "/home/buildadm/Ocean-Data-Map-Project/plotting/plotter.py", line 51, in prepare_plot
    self.load_data()
  File "/home/buildadm/Ocean-Data-Map-Project/plotting/stick.py", line 67, in load_data
    self.timestamp = timestamp
NameError: name 'timestamp' is not defined
```

**To Reproduce**
Request any stick plot. Here's an example query generated by the navigator:
```
/api/v1.0/plot/?query=%7B%22dataset%22%3A%22giops_day%22%2C%22depth%22%3A%5B0%5D%2C%22endtime%22%3A2214820800%2C%22names%22%3A%5B%2260.1524%2C%20-29.0382%22%5D%2C%22plotTitle%22%3A%22%22%2C%22quantum%22%3A%22day%22%2C%22station%22%3A%5B%5B60.152442214380784%2C-29.038238525390625%2Cnull%5D%5D%2C%22type%22%3A%22stick%22%2C%22variable%22%3A%22magwatervel%22%7D&format=json
```
And one with `starttime`:
```
/api/v1.0/plot/?query=%7B%22dataset%22%3A%22giops_day%22%2C%22depth%22%3A%5B0%5D%2C%22endtime%22%3A2214820800%2C%22names%22%3A%5B%2259.4897%2C%20-29.3019%22%5D%2C%22plotTitle%22%3A%22%22%2C%22quantum%22%3A%22day%22%2C%22starttime%22%3A2214561600%2C%22station%22%3A%5B%5B59.489726035537075%2C-29.301910400390636%2Cnull%5D%5D%2C%22type%22%3A%22stick%22%2C%22variable%22%3A%22magwatervel%22%7D&format=json
```

**Comment by htmlboss** [comment]
There are two parts to this one: the python side, and the javascript side messing up the start time picker.

My fix for #673 should help with this one too.

**Comment by JustinElms** [comment]
No longer displaying stick plots - closing.
